test/data/css/cssWidthBrowserZoom.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<style>
+		html {
+			zoom: 1.1
```

Comment:
This should have a semicolon for consistency.

Reply:
Resolved ✔️